camp/pom.xml

```diff
@@ -70,7 +50,7 @@
                 <configuration>
                     <archive>
                         <manifest>
-                            <mainClass>ch.bfh.zodnrk.camp.Application</mainClass>
+                            <mainClass>ch.bfh.zodnrk.camp.DemoApplication</mainClass>
```

Comment:
Should probably not be called 'DemoApplication'.
```suggestion
                            <mainClass>ch.bfh.zodnrk.camp.Application</mainClass>
```

Reply:
Thanks. Fixed as well.